### PR TITLE
Merge pull request #1 from fossasia/gh-pages

### PR DESCRIPTION
**Pull request by abhinavraj0123456789**
Added my photo

Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by the-codingninja** [comment]
Your pull request contains nothing please make a valid pull request

**Comment by anshumanv** [comment]
You need to add your details too, refer project's readme on `how to add a student`.

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please add your details as specified in readme and upload image in the correct path.

**Comment by abhinavraj0123456789** [comment]
Please merge it with the base branch

**Comment by abhinavraj0123456789** [comment]
How to add my details?

**Comment by anshumanv** [comment]
Follow - https://github.com/fossasia/gci17.fossasia.org#adding-a-student

**Comment by abhinavraj0123456789** [comment]
Can you please merge my photo to the master branch ?

**Comment by abhinavraj0123456789** [comment]
Please merge the photo which I have added, to the base branch.

**Review by isuruAb — COMMENTED** [review]
Hello @abhinavraj0123456789 ,Your Image size should be 240 x 240 [w x h]

**Comment by abhinavraj0123456789** [comment]
Yes, the size of the image is 240 x 240

**Review by the-codingninja — CHANGES_REQUESTED** [review]
You should follow readme on how to add a student

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Comment by abhinavraj0123456789** [comment]
What are the conflicts? @ms10398

**Comment by the-codingninja** [comment]
Check you could see resolve conflicts button you have to click on it and resolve conflicts and provide a preview link

**Comment by abhinavraj0123456789** [comment]
Is it ok now? @ms10398

File: _data/students.yml [hunk 1]
```diff
@@ -95,9 +95,14 @@
   github: aryankedare
   image: aryankedare.jpg
   twitter: KedareAryan
-  
+- name: Abhinav Raj
```
Comment (akshatnitd) [inline _data/students.yml]: Leave a blank line before it

File: _data/students.yml [hunk 2]
```diff
@@ -95,9 +95,14 @@
...
+  github: abhinavraj0123456789
+  image: AbhinavRaj.jpg
+  twitter: hardwired_raj
+
 - name: Samyak Jain
   github: samyakbvs
   image: samyak.png
   twitter: Samyakbvs
-  facebook: Samyak Jain 
+  facebook: Samyak Jain
```
Comment (akshatnitd) [inline _data/students.yml]: Leave a blank line at the end of the file

**Review by akshatnitd — CHANGES_REQUESTED** [review]
You have added the photo twice. Kindly remove the photo present at the root of the repository. Also, presently the dimensions of the photo is 180x240, kindly submit a photo with dmension 240x240.

**Comment by abishekvashok** [comment]
Pls update your fork. Remove unwanted commits - drop them, rebase properly and try to squash the commits down.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
@abhinavraj0123456789 You now have 3 images in the commit, out of which 2 of them are in incorrect locations. Kindly remove them from your PR.
Use ```git rm <file-name>```

File: _data/students.yml [hunk 3]
```diff
@@ -101,14 +101,21 @@
...
+  country: India
+  github: abhinavraj0123456789
+  image: AbhinavRaj.jpg
+  twitter: hardwired_raj
 
 - name: Samyak Jain
   github: samyakbvs
   image: samyak.png
   twitter: Samyakbvs
+  facebook: Samyak Jain
```
Comment (akshatnitd) [inline _data/students.yml]: @abhinavraj0123456789 Extra line added. Kindly remove it.

**Review by akshatnitd — CHANGES_REQUESTED** [review]

File: _data/students.yml (hunk 3 above)
Comment (the-codingninja) [reply _data/students.yml]: @abhinavraj0123456789 Just remove this line and your PR is ready to merge

File: _data/students.yml [hunk 4]
```diff
@@ -101,14 +101,21 @@
...
   image: samyak.png
   twitter: Samyakbvs
+  facebook: Samyak Jain 
   facebook: Samyak Jain
   
 - name: Tushar Verma
   github: 2coolife
   image: 2coolife.jpg
-  twitter: 2coolife
+  twitter: 2coolife
```
Comment (the-codingninja) [inline _data/students.yml]: Add a blank line after this @abhinavraj0123456789

**Review by the-codingninja — CHANGES_REQUESTED** [review]

File: _data/students.yml (hunk 3 above)
Comment (akshatnitd) [reply _data/students.yml]: @abhinavraj0123456789 This facebook link of Samyak Jain is added twice, kindly update the PR

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please remove the extra Facebook link and update the PR

**Comment by akshatnitd** [comment]
@abhinavraj0123456789 Your last commit is empty, kindly recheck. Also,your image dimensions are 180x240, resize it to 240x240.

**Comment by the-codingninja** [comment]
Travis is failing as the image is not of 240X240 dimension

**Comment by ParthS007** [comment]
@abhinavraj0123456789 please resolve conflicts and do the suggested changes

**Review by abhinavk96 — CHANGES_REQUESTED** [review]
@abhinavraj0123456789 can you please remove the duplicate line so that it can be merged. Thanks!

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@abhinavraj0123456789  You have to only resolve conflicts, Please do it and close other PR related to it

**Comment by ParthS007** [comment]
closing this PR due to inactivity 👍

**Pull request closed without merging**